index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+
+        .crnr-img {
+            position: relative;
+            left: 12%;
+            top: -60px;
+            width: 150px;
+
+        }
+        .buttom h1{
+    margin: 0 !important;
+}
+.buttom ul{
+    margin: 0 !important;
+}
+    </style>
+</head>
+
+<body>
+    <nav class="container">
+        <h1>Hasenat</h1>
+        <img src="images/Icon - Menu.png" alt="">
+    </nav>
+    <div class="content-container">
+        <h1>
+            Hi There, <br>I'm Hasenat
+        </h1>
+        <h3>I am a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+        <ul class="social-media">
+            <li><img src="images/github.png" alt=""></li>
+            <li><img src="images/linkedin.png" alt=""></li>
+            <li><img src="images/Vector.png" style="margin-top: 12px;" alt=""></li>
+            <li><img src="images/twitter.png" alt=""></li>
+            <li><img src="images/medium.png" alt=""></li>
+        </ul><br><br>
+        <img class="corner-img" src="images/Group.png" alt="">
+
+    </div>
+
+    <div class="recent-work-container">
+        <h1>My Recent Works</h1>
+        <hr>
+    </div>
+    <div class="multi-post-stories">
+        <img src="images/Img Placeholder.png" alt="">
+        <h1>Multi-Post Stories</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="container-list">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <button class="see-project-btn">See Project</button>
+    </div>
```

Comment:
Kindly enclose all of works section within  a `section` tag since they make up a section of the website.